src/seedu/addressbook/AddressBook.java

```diff
@@ -364,24 +387,27 @@ private static void loadDataFromStorage() {
      * @param userInputString  raw input from user
      * @return  feedback about how the command was executed
      */
+    public enum commandType {
+    	ADD, FIND, LIST, DELETE, CLEAR, HELP, EXIT
+    }
     private static String executeCommand(String userInputString) {
         final String[] commandTypeAndParams = splitCommandWordAndArgs(userInputString);
         final String commandType = commandTypeAndParams[0];
         final String commandArgs = commandTypeAndParams[1];
         switch (commandType) {
```

Comment:
Note that comandType here refers to the String variable declared at line 395 and not the enum type. This is a good example of why the coding standards are important. It helps avoids having name clashes.